### ISSUE TITLE
invalid continuation byte

### ISSUE THREAD

**Issue by arknoll** [laspy/laspy#148]
Have you seen this before?

running:

`with laspy.open(lasFile) as file:`

receiving error:
```
  File "/home/ubuntu/processing/CloudProcessing/process/utilities/lasUtilities.py", line 483, in chunk_split
    with laspy.open(lasFile) as file:
  File "/home/ubuntu/miniconda3/envs/cloudprocessing/lib/python3.8/site-packages/laspy/lib.py", line 110, in open_las
    return LasReader(stream, closefd=closefd, laz_backend=laz_backend)
  File "/home/ubuntu/miniconda3/envs/cloudprocessing/lib/python3.8/site-packages/laspy/lasreader.py", line 40, in __init__
    self.header = LasHeader.read_from(source)
  File "/home/ubuntu/miniconda3/envs/cloudprocessing/lib/python3.8/site-packages/laspy/header.py", line 479, in read_from
    stream.read(SOFTWARE_IDENTIFIER_LEN).rstrip(b"\0").decode()
UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 16-17: invalid continuation byte
```

**Comment by tmontaigu** [comment]
No,

It seems like the Software identifier written in the header of the file is not encoded in ASCII or UTF-8.

**Comment by hobu** [comment]
> It seems like the Software identifier written in the header of the file is not encoded in ASCII or UTF-8.

Common on really old data.

**Comment by arknoll** [comment]
What do you think about this https://github.com/laspy/laspy/pull/149 instead of hard erroring out?

**Comment by tmontaigu** [comment]
Fixed by #149
